doc/api/module.md

```diff
@@ -1048,6 +1048,47 @@ returned object contains the following keys:
 * columnNumber: {number} The 1-indexed columnNumber of the
   corresponding call site in the original source
 
+## TypeScript support
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.0 - Early development
+
+It is possible to execute TypeScript files by by setting the experimental
```

Comment:
Again, I don't think this should say 'TypeScript' files, but rather 'files with type annotations'